src/main/java/lotto/domain/LottoRank.java

```diff
@@ -0,0 +1,46 @@
+package lotto.domain;
+
+import java.util.Arrays;
+import java.util.Optional;
+
+public enum LottoRank {
+    LOSE(0, 0),
+    ONE(6, 2000000000),
+    THREE(5, 1500000),
+    FOUR(4, 50000),
+    FIVE(3, 5000)
+    ;
+
+    private final int matchCount;
+    private final long winAmount;
+
+    LottoRank(int matchCount, long winAmount) {
+        this.matchCount = matchCount;
+        this.winAmount = winAmount;
+    }
+
+    public int getMatchCount() {
+        return matchCount;
+    }
+
+    public long getWinAmount() {
+        return winAmount;
+    }
+
+    public static LottoRank getEnum(int matchCount) {
+        if(isLose(matchCount)) {
+            return LOSE;
+        }
+        Optional<LottoRank> find = Arrays.stream(LottoRank.values())
+                .filter(lottoRank -> lottoRank.getMatchCount() == matchCount)
+                .findFirst();
```

Comment:
프로그래밍 요구사항을 확인해보시면 **indent(인덴트, 들여쓰기) depth를 2를 넘지 않도록 구현한다. 1까지만 허용한다.**라는 규칙이 있습니다. for문안에 if문이 들어가있어서 2depth가 되었네요 ㅠ.ㅠ 확인부탁드립니다.

```suggestion
               for (LottoRank lottoRank : LottoRank.values()) {
                  if (lottoRank.getMatchCount() == matchCount) {
                      find = Optional.of(lottoRank);
                      break;
                  }
        }
```